app/assets/sweetalert.css

```diff
@@ -0,0 +1,932 @@
+body.stop-scrolling {
```

Comment:
I want to add LESS and SCSS compiling to gulpfile soon but in the meantime if you want to add CSS files please put it in a css directory within assets.